package.json

```diff
@@ -414,5 +455,7 @@
     "vscode-languageclient": "3.5.0",
     "vscode-languageserver-protocol": "3.5.0",
     "vscode-languageserver-types": "3.5.0"
-  }
+  },
+  "requiredDotNetCoreSDK": "2.1.0",
```

Comment:
This is just what I will use in the code to determine what is the minimum SDK version we need. It would not install the net core.

Reply:
Actually, dotnet _can_ be auto-installed as a dependency: https://www.npmjs.com/package/dotnet-sdk-2.1

But then, each extension will have its own private version of dotnet, unless it was already installed globally.

Reply:
This is interesting.. How big is the payload, do you know?

Reply:
It says it will only be available for the scripts in that package. I don't think user would be able to run dotnet commands in the terminal. Isn't it better to have them install it globally?

Reply:
I have just played with it for a bit in my WSL. It is installed in a shared location in the current user home directory. It is actually documented in the package readme. It is not added to the path automatically, but you can run dotnet commands from that directory. The payload... well... it is regular dot net installation size:

```
sergei@:~/.net/2.1/node_modules/dotnet-sdk-2.1-linux-x64$ du -hs
327M    .
```

Reply:
There is nothing wrong with having them install it locally, I was just thinking how can we eliminate additional friction...
So Terminal won't work, as Swikriti mentioned? Lets start with manual install for now and we can revisit it later, I don't think it will be a breaking change...

Reply:
The terminal will work. You just need to change the directory, or add it to the PATH, etc..

Reply:
I feel this requires more research and testing and can be part of the next release. Otherwise we would be cutting it too close for Ignite. Yes, I think it should not be a breaking change and can come in next release. Will come up with proper plan by then.

Reply:
I agree, sounds good!
I think changing directory to where dotnet is, is not really a good option - I'd rather use the full path, but lets think about it later. It seems like it will be only one install for multiple extensions, if they all use the same version.